verifier/src/qe_identity.rs

```diff
@@ -136,21 +136,22 @@ impl QeIdentity {
         sgx_attributes_t { flags, xfrm }.into()
     }
 
-    fn verify(&self, time: DateTime) -> Result<(), Error> {
+    fn verify(&self, time: Option<DateTime>) -> Result<(), Error> {
```

Comment:
Note if you do `impl Into<Option<DateTime>>` for the public functions, I would probably leave these private ones as `Option<DateTime>`